File: Python/ExampleTests/production/Address.py
from .SimpleText import SimpleText


class Address:
    def __init__(self, street: SimpleText, city: SimpleText,
                 state: SimpleText, zip: SimpleText):
        self.street = street
        self.city = city
        self.state = state
        self.zip = zip

    def __eq__(self, other):
        if not isinstance(other, Address):
            return NotImplemented
        return (self.street, self.city, self.state, self.zip) == \
               (other.street, other.city, other.state, other.zip)

    def __hash__(self):
        return hash((self.street, self.city, self.state, self.zip))

    def __str__(self):
        return f'Address{{{self.street}, {self.city}, {self.state}, {self.zip}}}'

File: Swift/ExampleTests/production/Address.swift
public struct Address {
    public let street: SimpleText
    public let city: SimpleText
    public let state: SimpleText
    public let zIP: SimpleText

    public init(street: SimpleText, city: SimpleText, state: SimpleText, zIP: SimpleText) {
        self.street = street
        self.city = city
        self.state = state
        self.zIP = zIP
    }
}
